export/export.go

```diff
@@ -501,3 +515,10 @@ func (s *Service) applyPods(ctx context.Context, r *ResourcesForImport, eg *util
 	}
 	return nil
 }
+
+// isSystemPriorityClass returns whether the given name of PriorityClass is prefixed with `system-` or not.
+// The prefix `system-` is reserved by Kubernetes and cannot be used in the name of PriorityClass.
```

Comment:
Sorry, my wording is not good 😓

```suggestion
// The prefix `system-` is reserved by Kubernetes and users cannot create PriorityClass that name is prefixed with `system-`.
```

Reply:
@sanposhiho How is this?
```
The `system-` prefix is reserved by Kubernetes, and users cannot create a PriorityClass with such a name.
```